### ISSUE TITLE
This is my client's skid lol

### ISSUE THREAD

**Issue by freezeOP** [Mese3d/Skiddle-leaked#1]
Skiddle is lunacia skid

but its old.
Im also retarded

**Comment by ElectroDaze** [comment]
yup he skidded 15 skidded clients and put them all together into 1 big skid

**Comment by mqivaUwU** [comment]
Skiddley dop dop skid skid

**Comment by freezeOP** [comment]
> yup he skidded 15 skidded clients and put them all together into 1 big skid

yea, he is best skidder lol

**Comment by mqivaUwU** [comment]
ty
________________________________

**Comment by freezeOP** [comment]
no problem :>>>> 
Thank you for skid my client :)))))))))))))))))
